### PR TITLE
RB-1310: Semantic UI update

### PR DESCRIPTION
**Pull request by LawrieR**

### CODE REVIEW

**Comment by craig-rainbird** [comment]
package.json will need upping and you've accidentally included package-lock file.

File: source/directives/pluralNumbers.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+									//perform backspace action
+									scope.inputValue = scope.inputValue.substring(0, scope.inputValue.length -1);
+									scope.answers.currentValue = scope.inputValue;
+							} else {
+								scope.answers.splice(scope.answers.length-1, 1);
+								scope.answers.currentValue = '';
+								e.preventDefault();
+							}
+							scope.$apply();
+						} else if (e.key == '.') {
```
Comment (craig-rainbird) [inline source/directives/pluralNumbers.js]: Is this needed?

File: source/goalList/agentGoalListController.js [hunk 2]
```diff
@@ -53,6 +53,7 @@ angular.module('rbAgent')
     }
 
     function handleError(err) {
+    	console.error(err);
```
Comment (craig-rainbird) [inline source/goalList/agentGoalListController.js]: Did you mean to leave this in?

Comment (LawrieR) [reply source/goalList/agentGoalListController.js]: Yea

File: source/tryGoal/component/tryGoalController.js [hunk 3]
```diff
@@ -9,6 +9,8 @@ function($scope, agentMemory, $compile, $stateParams, config, GoalAPI, ConfigAPI
     $scope.yolandaUrl = ApiConfig.getConfig().url;
     $scope.tryGoal = agentMemory.tryGoal;
 
+    $scope.test = 'none';
```
Comment (craig-rainbird) [inline source/tryGoal/component/tryGoalController.js]: Is this needed?

File: source/tryGoal/component/tryGoalController.js [hunk 4]
```diff
@@ -335,6 +265,14 @@ function($scope, agentMemory, $compile, $stateParams, config, GoalAPI, ConfigAPI
                 }
                 responseObject.push(temp);
             } else { //Second form
+				if (question.answer.selection.currentValue) {  //currentValue is used by some plural inputs where having it's value in the selection array is awkward.
+					question.answer.selection.push(question.answer.selection.currentValue);
+					delete question.answer.selection.currentValue;
+				}
+				if ('typeof', typeof (question.answer.selection) == 'string') {
```
Comment (craig-rainbird) [inline source/tryGoal/component/tryGoalController.js]: This doesn't look quite right to me. Was the first 'typeof' meant to be checking something?

File: thirdparty/SemanticUI-Angular/.travis.yml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+  - 0.10
```
Comment (craig-rainbird) [inline thirdparty/SemanticUI-Angular/.travis.yml]: Is the node version important?

File: thirdparty/SemanticUI-Angular/examples/accordion/directives.html [hunk 6]
```diff
@@ -0,0 +1,228 @@
+<!DOCTYPE html>
```
Comment (craig-rainbird) [inline thirdparty/SemanticUI-Angular/examples/accordion/directives.html]: Do we have to bring in all the examples?

Comment (LawrieR) [reply thirdparty/SemanticUI-Angular/examples/accordion/directives.html]: No, it's just a clone if their git repo. I had to do a one-line change to their stuff :( I can delete them if you want.

**Pull request merged**